functests/5_latency_testing/latency_testing.go

```diff
@@ -214,7 +214,7 @@ func getValidValuesTests(toolToTest string) []latencyTest {
 	if toolToTest == oslat {
 		testSet = append(testSet, latencyTest{testRun: "true", testRuntime: "10", testMaxLatency: "1", oslatMaxLatency: guaranteedLatency, outputMsgs: []string{success}, toolToTest: toolToTest})
 		testSet = append(testSet, latencyTest{testRun: "true", testRuntime: "10", oslatMaxLatency: guaranteedLatency, outputMsgs: []string{success}, toolToTest: toolToTest})
-		testSet = append(testSet, latencyTest{testRun: "true", testRuntime: "10", testMaxLatency: guaranteedLatency, testCpus: "1", outputMsgs: []string{skip, skipOslatCpuNumber}, toolToTest: toolToTest})
+		testSet = append(testSet, latencyTest{testRun: "true", testRuntime: "10", testMaxLatency: guaranteedLatency, testCpus: "0", outputMsgs: []string{skip, skipOslatCpuNumber}, toolToTest: toolToTest})
```

Comment:
however requesting 0 cpus is misleading for another reasons. Maybe ask 2?

Reply:
it is either 0 or 1 are the invalid for oslat, as it works fine with at least 2 CPUS

Reply:
following the commit: https://github.com/openshift-kni/performance-addon-operators/pull/891/commits/0f197be30938d8e25b7556eab7667349614f93e8
this test set is no longer relevant. Will address an alternative in a later PR when managing the PP within the suite.